CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ PowerModels.jl Change Log
 ### Staged
 - Refactor connected_components to accept arbitrary edge-types with `edges` kwarg
 - Fixes bug in warnings inside storage constraints
+- Add Bool to allowed data types in json parse of MultiConductorVector/Matrix
+- Add handling of Inf/NaN values in JSON parser
```

Comment:
I think this second line should also qualify with "in MultiConductorVector/Matrix"